tools/molecule.spec

```diff
@@ -0,0 +1,59 @@
+%global srcname molecule
+%global pypiname molecule
+%global pkgname python3-%{srcname}
+%global forgeurl https://github.com/ansible-community/%{srcname}
+
+Name:    %{pkgname}
+Version: 0.0.0
+Release: 1%{?dist}
+Summary: ...
+%forgemeta
+URL:       %{forgeurl}
+Source:    %{forgesource}
+License:   BSD
+BuildArch: noarch
+
+BuildRequires: python3-devel
+BuildRequires: python3-wheel
+BuildRequires: python3-tox
+# BuildRequires: python3-tox-current-env
```

Comment:
I think the only thing truly necessary here is PEP 517 aware macros. Others are pulled in automatically.

Reply:
The generate_buildrequires section is not used here anymore, so the buildrequires are needed. However, using the pyproject-rpm-macros like this is not really supported.